src/RecoveryServices/RecoveryServices.SiteRecovery.Test/ScenarioTests/A2A/AsrA2ATests.ps1

```diff
@@ -549,6 +549,143 @@ function Test-ReplicateProximityPlacementGroupVm{
 		Assert-NotNull($pe.providerSpecificDetails.RecoveryProximityPlacementGroupId)
 }
 
+<#
+.SYNOPSIS 
+    Test VMNicConfig new parametersets
+#>
+
+function Test-VMNicConfig{
+   param([string] $seed ='330')
+        $primaryPolicyName = getPrimaryPolicy
```

Comment:
Not required. You're using $policyName later.

Reply:
done